Automattic/vip
# Fix prompt handling causing duplicate messages to be shown for export sql and dev-env sync

## Description

Fixes an issue with prompt handling that introduces a regression with the promptless flow.

The bug is as per below:

```
$ vip export sql @0000.production
✓ Latest backup found with timestamp 2023-11-06T10:54:14.000Z
Exporting database backup with timestamp 2023-11-06T10:54:14.000Z
✓ Preparing for backup download
✓ Creating backup copy
⠦ Checking if there's enough storage <-- shouldn't repeat
○ Requesting download link <-- shouldn't repeat
○ Downloading file <-- shouldn't repeat
✓ Checking if there's enough storage
✓ Requesting download link
✓ Downloading file - 100.00% (52.34 KB/52.34 KB)
File saved to /some/path/some-backup.sql.gz
```

This PR removes the lines that say 'shouldn't repeat'

## Pull request checklist


## New release checklist


## Steps to Test

1. Check out PR.
2. Make sure you have a dev-env running - see [here](https://docs.wpvip.com/vip-local-development-environment/)
3. Run `npm run build:watch`
4. Run `./dist/bin/vip-dev-env-sync-sql.js @0000.production --slug=local-env`. Replace 0000 with your app. Bug should no longer be there.
5. Run `./dist/bin/vip-export-sql.js @0000.production`. Bug should no longer be there.

For regression test - do step 4 and 5 again, but force the prompt to always show by overriding the conditionals at [here](https://github.com/Automattic/vip-cli/blob/4c8127eff89a93ca3ac8bb6f05a73cc7ce17664f/src/lib/backup-storage-availability/backup-storage-availability.ts#L124-L124), [here](https://github.com/Automattic/vip-cli/blob/4c8127eff89a93ca3ac8bb6f05a73cc7ce17664f/src/lib/backup-storage-availability/backup-storage-availability.ts#L151-L151), and [here](https://github.com/Automattic/vip-cli/blob/4c8127eff89a93ca3ac8bb6f05a73cc7ce17664f/src/lib/backup-storage-availability/backup-storage-availability.ts#L97-L97).
